app/views/shopping_list/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<p style="color: green"><%= notice %></p>
+
+<section class = 'd-flex flex-column mb-2 align-items-center'>
+<div class="section-title inventory_foods">
+  <h1>
+    Shopping List
+  </h1>
+  <div class="shopping_details">
+    <div>
+    <p>Amount of food to buy: <span><%= @shopping_list.length %></span> </p>
+    <p>Total value of food needed: <span>$ <%= @total_price %></span> </p>
+    </div>
+
+    <div>
+      <p>Recipe: <%= link_to "Recipe #{@recipe_to_find}", "/recipes/#{@recipe_to_find}", method: :get, class:''%></p>
+      <p>Recipe: <%= link_to "Inventory  #{@inventory_to_find}", "/inventories/#{@inventory_to_find}", method: :get, class:''%></p>    
```

Comment:
[REQUIRED] Kindly update this because I feel that you wanted to write inventory not recipe